src/components/common/calendar/CalendarModal.js

```diff
@@ -22,7 +22,8 @@ export default function CalendarModal(props) {
       time: values.time.format('h:mm A'),
       type: 'success',
     };
-    axiosWithAuth(idToken)
+    //TO-DO: Implement Auth0
+    axiosWithAuth('hi')
```

Comment:
Better to swap these `axiosWithAuth` instances out with plain old `axios` for now than to pass an incorrect variable to our `axiosWithAuth` function!

We still won't be able to receive data from the BE until our `authRequired` middleware is removed from endpoints, but moving to `axios` until we can fix `axiosWithAuth` to work with Auth0 in the next PR will prep us for that eventuality.